### PR TITLE
SAK-499 article changelog method

### PR DESCRIPTION
**Pull request by rafalkanski**
@filipgolonka @bartosz-kahl 

**DONE**
- `findByChangelog` for `ArticleRepository`

Parameters for narrowing changelog in next `PR`

### CODE REVIEW

File: features/bootstrap/Snt/Capi/ArticleRepositoryContext.php [hunk 1]
```diff
@@ -148,4 +160,48 @@ public function thereAreArticlesForPublicationInApi($publicationId, PyStringNode
...
+    /**
+     * @Then I should get articles changelog for :arg1 publication with content from API
+     *
+     * @param string $publicationId
+     *
+     * @throws PhpUnitExpectationFailedException
+     */
+    public function iShouldGetArticlesChangelogForPublicationWithContentFromApi($publicationId)
+    {
+        PhpUnit::assertEquals($this->articleChangelogFromApi[$publicationId], $this->articlesChangelog[$publicationId]);
```
Comment (filipgolonka) [inline features/bootstrap/Snt/Capi/ArticleRepositoryContext.php:205]: uparty :D

**Comment by filipgolonka** [comment]
:+1:

File: features/bootstrap/Snt/Capi/ArticleRepositoryContext.php (hunk 1 above)
Comment (rafalkanski) [inline features/bootstrap/Snt/Capi/ArticleRepositoryContext.php:205]: wsparcie od 5.6 wiec wy&(*&(lem !

Comment (rafalkanski) [inline features/bootstrap/Snt/Capi/ArticleRepositoryContext.php:205]: dlatego dodalem testy composer install dry run 💃

File: features/bootstrap/Snt/Capi/ArticleRepositoryContext.php [hunk 2]
```diff
@@ -148,4 +160,48 @@ public function thereAreArticlesForPublicationInApi($publicationId, PyStringNode
...
+     */
+    public function iAskForArticlesChangelogForPublicationUsingArticleRepository($publicationId)
+    {
+        $findParameters = FindParameters::createForPublicationId($publicationId);
+
+        $this->articlesChangelog[$publicationId] = $this->articleRepository->findByChangelog($findParameters);
+    }
+
+    /**
+     * @Then I should get articles changelog for :arg1 publication with content from API
```
Comment (barkahl) [inline features/bootstrap/Snt/Capi/ArticleRepositoryContext.php]: arg1

**Comment by barkahl** [comment]
👍

File: features/bootstrap/Snt/Capi/ArticleRepositoryContext.php (hunk 2 above)
Comment (rafalkanski) [inline features/bootstrap/Snt/Capi/ArticleRepositoryContext.php]: ale przynajmniej widac ze generowane za posred. BDD 💃

**Pull request merged**